src/components/DisplayDate.jsx

```diff
@@ -1,12 +1,19 @@
-import { Center, Title } from "@mantine/core";
+import { Center, MediaQuery, Title } from "@mantine/core";
 import { DISPLAYDATE } from "../constants/displayDate";
 
 const DisplayDate = () => {
   return (
     <Center>
-      <Title radius={"sm"} mb={"xl"} pt={"sm"} color={DISPLAYDATE}>
-        {new Date().toDateString()}
-      </Title>
+      <MediaQuery smallerThan={"md"} styles={{ display: "none" }}>
```

Comment:
The best practice is that we should have a separate file for styles of each component and styles code should live in that file.

This should be addressed as a tech debt item. When you will be refactoring File/Folder Structure.